Meadowlark-js/packages/meadowlark-core/src/model/SuperclassInfo.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { DocumentIdentity } from './DocumentIdentity';
+
+/**
+ * The optional superclass information for a DocumentInfo. Applies only to documents that are subclasses,
+ * providing superclass identity information. (Note the ODS/API allows only a single level of subclassing.)
+ */
+export type SuperclassInfo = {
```

Comment:
A rename of Assignable, and alignment of equivalent property names to be like those in DocumentInfo